docs/guide/migration.md

```diff
@@ -14,6 +14,19 @@ Jest has their [globals API](https://jestjs.io/docs/api) enabled by default. Vit
 
 If you decide to keep globals disabled, be aware that common libraries like [`testing-library`](https://testing-library.com/) will not run auto DOM [cleanup](https://testing-library.com/docs/svelte-testing-library/api/#cleanup).
 
+**Module mocks**
+
+When mocking a module in Jest, if the factory argument returned a primitive, the mock would implicitly mock the default export. In Vitest, the factory argument has to return an object where each export is explicitly defined. For example, the following `jest.mock` would have to be updated as follows:
```

Comment:
I don't think it has to be a primitive. Everything returned from a mock was available on `default`, since it is default CJS behaviour, no?

Reply:
I'm not actually sure, I assumed looking at the jest docs that a factory returning an object would result in named exports, and or require default explicitly be defined (if you were wanting default?) I can play with jest to better understand.

Reply:
You are indeed correct, will post an update shortly